.github/workflows/sync_translations.yml

```
@@ -34,16 +29,44 @@ jobs:
 
       - name: Credential Prep
         run: |
-          mkdir ~/src
-          echo "${{ secrets.CROWDIN_API_KEY }}" > ~/src/crowdin_key.txt
+          echo "${{ secrets.CROWDIN_API_KEY }}" > ./tools/localization/.env
         shell: bash
 
+      - name: Setup nodejs and yarn
+        run: |
+          sudo apt install npm -y
+          sudo npm install yarn -g
```

I think if we do this it is cross-platform, [18 is already "current" and will be LTS in a couple months](https://nodejs.org/en/about/releases/) so best to try now?, and yarn (1.2.x) is installed on the machines already (for [macos](https://github.com/actions/virtual-environments/blob/main/images/macos/macos-12-Readme.md) and [ubuntu](https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2204-Readme.md) virtual environments)

```suggestion
      - uses: actions/setup-node@v3
        with:
          node-version: 18
```

---

This is better way to setup nodejs. Thanks